test/quantization/test_quantized_op.py

```diff
@@ -2128,6 +2130,129 @@ def test_qlinear(self, batch_size, input_channels, output_channels,
         self.assertEqual(Y_fp32, Y_fp32_ref,
                          msg="torch.ops.quantized.linear_dynamic results are off")
 
+class TestDynamicQuantizedRNNOp(TestCase):
+    """Tests the correctness of the dynamic quantized lstm/gru."""
+    @given(
+        num_batches=st.integers(1, 4),
+        input_size=st.integers(16, 32),
+        hidden_size=st.integers(4, 8),
+        num_directions=st.integers(1, 2),
+        per_channel_quant=st.booleans())
+    @override_qengines
+    def test_qlstm(self, num_batches, input_size, hidden_size,
+                   num_directions, per_channel_quant):
+
+        def _get_rnn_inputs(seq_len, num_batches, input_size, hidden_size, num_directions):
+            # For Input (seq_len, batch, input_size)
+            X = torch.randn(seq_len, num_batches, input_size)
+            # TODO: Change to reduce_range=True once support is enabled
+            s, z = _calculate_dynamic_qparams(X, torch.quint8, reduce_range=False)
+            Xq = torch.quantize_per_tensor(X, s, z, torch.quint8)
+
+            # For H and C: (num_layers(1) * num_directions, batch, hidden_size)
+
+            if num_directions == 1:
+                H = torch.randn(num_directions, num_batches, hidden_size)
+                C = torch.randn(num_directions, num_batches, hidden_size)
+            else:
+                H = torch.zeros(num_directions, num_batches, hidden_size)
+                C = torch.zeros(num_directions, num_batches, hidden_size)
+
+            s, z = _calculate_dynamic_qparams(H, torch.quint8, reduce_range=False)
+            Hq = torch.quantize_per_tensor(H, s, z, torch.quint8)
+            s, z = _calculate_dynamic_qparams(C, torch.quint8, reduce_range=False)
```

Comment:
Same here

Reply:
Thanks, have made all the changes requested.